src/ddl/dashboard_signal_status.sql

```diff
@@ -0,0 +1,24 @@
+/*
+
+This table stores the status for signals in the public signal dashboard.
+
++-------------------+---------+------+-----+---------+-------+
+| Field             | Type    | Null | Key | Default | Extra |
++-------------------+---------+------+-----+---------+-------+
+| signal_id         | int(11) | NO   | PRI | NULL    |       |
+| date              | date    | NO   | PRI | NULL    |       |
+| latest_issue_date | date    | NO   |     | NULL    |       |
+| latest_data_date  | date    | NO   |     | NULL    |       |
++-------------------+---------+------+-----+---------+-------+
+
+*/
+
+CREATE TABLE `dashboard_signal_status` (
+  `signal_id` int(11) NOT NULL,
+  `date` date NOT NULL,
+  `latest_issue_date` date DEFAULT NULL,
+  `latest_data_date` date DEFAULT NULL,
```

Comment:
perhaps im missing something, but i am not clear on the distinction between `date` and `latest_data_date` in this

Reply:
My interpretation was:
* `date` - date this row was generated
* `latest_issue_date` - max issue
* `latest_data_date` - max(time_value) where issue==max_issue

Reply:
ah ok, that makes sense then.  perhaps we should come up with standardized terms?

Reply:
hmm yes, though part of the problem is that the signal dash is a bit meta -- a row in covidcast is about events that occurred on `time_value`, and was published on `issue`, whereas the "events" we're talking about in the signal dash are the publishing of other covidcast rows. we should either
* use existing covidcast terminology: (latest_issue_date -> latest_issue; latest_data_date -> latest_time_value) or perhaps (date -> issue, latest_issue_date -> signal_latest_issue, latest_data_date -> signal_latest_time_value)
* round up a bunch of stakeholders to revise our date terminology (iirc roni's not super fond of 'time_value' and prefers 'reference_date'), then migrate as necessary to use the same terms everywhere

Reply:
This is a really helpful discussion.

Since refactoring everything seems unlikely, I think it's a good idea to use terms mapping as closely as possible to the existing ones (for now). If the dashboard diverges enough we can rename.